src/api/converter.ts

```diff
@@ -926,12 +924,12 @@ function objectToBytes(obj: JSONObject): Uint8Array {
 }
 
 export const converter = {
-  fromMetadataMap: fromMetadataMap,
-  toClient: toClient,
-  toChangePack: toChangePack,
-  fromChangePack: fromChangePack,
-  toDocumentKeys: toDocumentKeys,
-  fromDocumentKeys: fromDocumentKeys,
-  objectToBytes: objectToBytes,
-  bytesToObject: bytesToObject,
+  fromMetadataMap,
+  toClient,
+  toChangePack,
+  fromChangePack,
+  toDocumentKeys,
+  fromDocumentKeys,
+  objectToBytes,
+  bytesToObject,
```

Comment:
It would be nice to add `object-shorthand` to the rules so it can be checked automatically
```
// .eslintrc.js
'object-shorthand': 'error',
```

https://eslint.org/docs/rules/object-shorthand